data/human/transport missions.txt

```diff
@@ -3582,3 +3582,732 @@ mission "Hauler VI cargo prototype 3"
+			`	"We are out of time. We need to reopen," says Mariel, pushing you toward the exit.`
+			`	"Wow, that's really intense," says Ale as everyone leaves. "We really deserve a vacation. Hey <first>, we have always wanted to go to Skymoot to see the dragons, do you think you could take us?"`
+			choice
+				`	"I would be happy to."`
+					goto happyto
+				`	"I should really be getting back to the war."`
+			`	"It's on the way but I can see you are busy. We will be in touch," she says, smiling.`
+				decline
+			label happyto
+			`	"Wonderful. We will get our stuff and meet you at the <ship>," she says, smiling.`
+				accept
+	on visit
+		dialog `You must wait for your escorts to reach this system.`
+	on complete
+		conversation
+			`As you bring the <ship> down to the spaceport you see a dragon flying nearby and point it out to your passengers who 'ooo' and 'ahh'. They can hardly contain their excitement as they leave the ship. Each one gives you a hug and wishes you good luck.`
+			`	"We will definitely be in touch," says Ale as they leave.`
+
+
+
+mission "Freedom Uprising 1"
+	landing
+	deadline 14
+	source
+		government "Syndicate"
+	destination "Moonshake"
+	to offer
+		has "Smuggler's Den, Part 2: done"
+		has "Journalism 5: done"
+	on offer
+		conversation
+			`	Back on your ship you notice a new message from Ale: "Hey. We are nearly done with the Syndicate Slavery project but it will still be a while for the exposition is ready to tour. In the meantime we have another project, but as usual it's time sensitive. Meet up with us on Moonshake soon or we will leave without you. Same place as before."`
+				accept
+
+
+
+mission "Freedom Uprising 2"
+	name "Road to Freedom"
+	landing
+	description "Take Ale, Beto and Bunker to <destination>."
+	deadline 14
+	source "Moonshake"
+	destination "Sunracer"
+	to offer
+		has "Freedom Uprising 1: done"
+		not "main plot completed"
+	passengers 3
+	on offer
+		conversation
+			`You find Ale and Bunker in the 'Common Ground Cafe' chatting with a young man in his mid-twenties. You notice they have backpacks and some cases with them. "There you are," says Bunker. "I was just telling them how you handled those pirate slavers."`
+			`	"For like the hundredth time," says Ale, rolling her eyes.`
+			`	"That's because it was epic. Aside from the thought of that pirate sensing the drones. If that's even what happened. It still gives me the chills.`
+			choice
+				`	"I haven't slept well since that day."`
+				`	"I think it was just a coincidence."`
+			`	"Whatever it was, it was weird. Ah, remember when we talked about liberating slave instead of filming them. Well this is Beto," he says, now more quietly.`
```

Comment:
```suggestion
			`	"Whatever it was, it was weird. Ah, remember when we talked about liberating slaves instead of filming them. Well this is Beto," he says, now more quietly.`
```